### DISCUSSION TITLE
Wrong example sentences for はぁ

### DISCUSSION THREAD

**Discussion by g3335491** [Jitendex/Jitendex#120] [Errors and Corrections]
JP: あの犬はしっぽが短い。

JP: かぜには大食、熱には小食。

JP: 母はぜったいに早起きはしない。

These sentences seems mistagged.

**Comment by stephenmk** [comment]
Thank you. I've [corrected the sentences](https://github.com/stephenmk/jmdict-tatoeba-sentence-linking/commit/da8e57106074048e85a9b25b57399b39098d1c5e) and they will be fixed in the next jitendex update.
